components/edit-collective/sections/Export.js

```diff
@@ -17,6 +18,198 @@ class Export extends React.Component {
     super(props);
   }
 
+  generateExports(collective, widgetCode) {
```

Comment:
Instead of a condition, i would create 2 different functions. This should be named "render" not "generate".

ie: `renderHostExports`, `renderCollectivesExport`.